packages/mobile/src/i18n.ts

```diff
@@ -41,7 +41,8 @@ const languageDetector = {
 const currencyInterpolator = (text: string, value: any) => {
   const key = value[1]
   const translations = currencyTranslations[i18n.language]
-  if (key in translations) {
+
+  if (translations && key in translations) {
```

Comment:
Don't really understand why `translations ` is not an object when the user changes the language to `es-419`. Maybe @annakaz knows.

Reply:
so when we rehydrate the redux store, we set i18n.language to be whatever the previous saved language is. since for other people it would be es-AR, indexing into currentTranslations['es-AR'] returns undefined. the root fix here should be fixing that rehydration, or ideally migrating